### PR TITLE
added basic sidekiq info to sat 6.10

### PR DESCRIPTION
**Pull request by Imaanpreet**

### CODE REVIEW

File: docs/satellite-configuration-tuning.rst [hunk 1]
```diff
@@ -291,6 +291,27 @@ Dynflow Tuning
...
+
+- Increase the number of workers, which is recommended.
+
+Below examples do increase 1 worker to 3 while remaining 5 threads/concurrency of each::
+
+ satellite-installer --foreman-dynflow-worker-instances 3    # optionally, add --foreman-dynflow-worker-concurrency 5
+
+Then check if there are three worker services::
+
+  systemctl -a | grep dynflow-sidekiq@worker-[0-9]
```
Comment (jhutar) [inline docs/satellite-configuration-tuning.rst:311]: Make this output narrower so each row does not span across multiple lines please.

File: docs/satellite-configuration-tuning.rst [hunk 2]
```diff
@@ -291,6 +291,27 @@ Dynflow Tuning
...
+
+From Satellite 6.8, we have a new dynflow service named “dynflow-sidekiq”. It is recommended to increase the sidekiq workers in order to scale the foreman tasking system for bulk concurrent tasks like multiple CV publish/promote , capsule sync etc. There are two options available:
+
+- Increase the number of threads by a dynflow worker (worker's concurrency). This has limited impact for values >5 due to ruby implementation of threads' concurrency.
+
+- Increase the number of workers, which is recommended.
+
+Below examples do increase 1 worker to 3 while remaining 5 threads/concurrency of each::
+
+ satellite-installer --foreman-dynflow-worker-instances 3    # optionally, add --foreman-dynflow-worker-concurrency 5
```
Comment (jhutar) [inline docs/satellite-configuration-tuning.rst]: Is one space indentation to format this as a code block sufficient, or do we need 4 spaces?

File: docs/satellite-configuration-tuning.rst [hunk 3]
```diff
@@ -291,6 +291,27 @@ Dynflow Tuning
 
 Dynflow is the workflow management system and task orchestrator which is built as a plugin inside Foreman and is used to execute the different tasks of Satellite in an out-of-order execution manner. Under the conditions when there are a lot of clients checking in on Satellite and running a number of tasks, the Dynflow can take some help from an added tuning specifying how many executors can it launch.
 
+Increase sidekiq workers
+========================
+
+From Satellite 6.8, we have a new dynflow service named “dynflow-sidekiq”. It is recommended to increase the sidekiq workers in order to scale the foreman tasking system for bulk concurrent tasks like multiple CV publish/promote , capsule sync etc. There are two options available:
```
Comment (jhutar) [inline docs/satellite-configuration-tuning.rst]: Please add some generic description of what the service is - does below make sense @adamruzicka?

```suggestion
From Satellite 6.8, we have a new Dynflow service named “dynflow-sidekiq” that performs tasks scheduled by Dynflow. Sidekiq workers can be grouped into various queues to ensure lots of tasks of one type will not block execution of tasks of other type. Currently there are these ques:

* xxx - used to execute tasks yyy
* zzz - used to execute tasks aaa

It is recommended to increase the sidekiq workers in order to scale the foreman tasking system for bulk concurrent tasks like multiple CV publish/promote , capsule sync etc. There are two options available:
```

File: docs/satellite-configuration-tuning.rst [hunk 4]
```diff
@@ -291,6 +291,27 @@ Dynflow Tuning
...
+
+- Increase the number of threads by a dynflow worker (worker's concurrency). This has limited impact for values >5 due to ruby implementation of threads' concurrency.
+
+- Increase the number of workers, which is recommended.
```
Comment (jhutar) [inline docs/satellite-configuration-tuning.rst:303]: Does it have some drawbacks? Memory concerns? What is the ideal count?

**Review by jhutar — CHANGES_REQUESTED** [review]

File: docs/satellite-configuration-tuning.rst (hunk 4 above)
Comment (Imaanpreet) [reply docs/satellite-configuration-tuning.rst:303]: not sure about the drawbacks and memory concerns. I have tested some test runs : REX with concurrency - 10 and 15 but unfortunately we don't have data in grafana due to some outage in infra lab. I am again testing to get some performance numbers.

File: docs/satellite-configuration-tuning.rst (hunk 3 above)
Comment (Imaanpreet) [reply docs/satellite-configuration-tuning.rst]: looks good to me, except these two : 
* xxx - used to execute tasks yyy
* zzz - used to execute tasks aaa

Rest I added. Thank you.

File: docs/satellite-configuration-tuning.rst (hunk 2 above)
Comment (Imaanpreet) [reply docs/satellite-configuration-tuning.rst]: one space indentation is required to make this code as block sufficient.

File: docs/satellite-configuration-tuning.rst (hunk 1 above)
Comment (Imaanpreet) [reply docs/satellite-configuration-tuning.rst:311]: done.

**Comment by jhutar** [comment]
This will be extended once we finish our testing

**Pull request merged**